src/main/java/it/gov/pagopa/pu/fileshare/connector/processexecutions/ExportFileServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.pu.fileshare.connector.processexecutions;
+
+import it.gov.pagopa.pu.fileshare.connector.processexecutions.client.ExportFileClient;
+import it.gov.pagopa.pu.fileshare.exception.custom.UnauthorizedFileDownloadException;
+import it.gov.pagopa.pu.fileshare.service.AuthorizationService;
+import it.gov.pagopa.pu.p4paauth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.p4paprocessexecutions.dto.generated.ExportFile;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ExportFileServiceImpl implements ExportFileService {
+
+  private final ExportFileClient client;
+
+  public ExportFileServiceImpl(ExportFileClient client) {
+    this.client = client;
+  }
+
+  @Override
+  public ExportFile getExportFile(Long exportFileId, Long organizationId, UserInfo loggedUser, String accessToken) {
+    boolean isLoggedUserAdmin = AuthorizationService.isAdminRole(
+      organizationId, loggedUser);
+    String operatorExternalUserId = null;
+
+    if(!isLoggedUserAdmin){
+      operatorExternalUserId = loggedUser.getMappedExternalUserId();
+    }
```

Comment:
This are business layer logic, not connector logic.
Move into ExportFileFacadeServiceImpl

Reply:
ps: this layer could be configured to introduce a cache! thus we cannot add business logic here